docs/eventing/samples/kafka/channel/README.md

```diff
@@ -151,3 +151,75 @@ Now you can see the events in the log of the `ksvc` using the command:
 ```
 kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container
 ```
+
+## Authentication against an Apache Kafka
+
+The KafkaSource supports TLS and SASL authentication methods.
+
+### TLS authentication
+
+
+To use TLS authentication you must create:
+
+* A CA certificate
+* A client certificate and key
+
+**NOTE:** Kafka channels require these files to be in `.pem` format. If your files are in a different format, you must convert them to `.pem`.
+
```

Comment:
not sure why but there seem to be a few parts with unnecessary empty lines, I'd remove them